android_splash_screen/android/app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright (C) 2021 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
```

Comment:
To my knowledge, I haven't seen copyright messages added to the .xml files

Reply:
@RedBrogdon What is the protocol here? I assumed these were the files you were referencing when saying some had headers missing.

Reply:
Any file that you've modified beyond what `flutter create` or the "New Android Project" templates spit out should have a copyright header. If you haven't modified this file, there's no need to add one. If you have, though, I believe it's correct to add a header.